### ISSUE TITLE
Interviewing previous GSoC and Outreachy contributors

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#671]
Once, @gitster suggested that we interview GSoC or Outreachy contributors. I think it might not be a good idea to do it during their contribution period, but I think it could be a good idea to interview them afterwards. Perhaps years after might be even better.

If they were mentored some years ago, I think it could be interesting to ask them questions like the following:

- How did their GSoC or Outreachy internship change their lives?
- Did they get better opportunities?
- Did they became better programmer?
- Did their Git skills improve?
- Are they still following the mailing list or the community somehow?

@sivaraam @jnareb and @mjaix what do you think about it?

**Comment by sivaraam** [comment]
Sounds like a good idea to me. It would indeed be an interesting one.

I wonder if we should keep this as a related section to "Developer Spotlight" but titled slightly differently to convey the difference in focus. May be we could call it something like "Alumni Retrospection" (or) something ?

**Comment by chriscool** [comment]
Yeah, I think it might be a good idea to use "Alumni Retrospection". I especially like that it's a bit different than usual and readers can easily notice it. On the other hand it might make it a bit difficult in the future to look for all the people we interviewed one way or another, but I think this is a small downside.

**Comment by jnareb** [comment]
I also think this is a good idea (perhaps under different name, like "From Mentee to Developer", but "Alumni Retrospection" is also a good suggestion).

We could however start with some GSoC and Outreachy statistics:
- how many students we had in each year, 
- was their contribution merged, 
- were they contributing in a year after the end of mentored time, 
- are they active now (authoring commits, doing reviews, or even active in discussion on git mailing list).

**Comment by chriscool** [comment]
@jnareb yeah, such statistics are very interesting, but I think it could be a separate article later. Feel free to work on it if you are interested in it. I don't think we really need it to start interviewing past contributors and interns though.

BTW we already have such information for previous GSoC in some SoC-*-Org-Application.md files, like SoC-2019-Org-Application.md, as Google used to ask us "What is your success/fail rate per year?" and we used to answer like this (from SoC-2019-Org-Application.md):

Here are the numbers of passed (p), failed (f) and retained (r) students (where "retained" means "still participating on the mailing list a year or more after their GSoC") for each year:

 - 2007: 2 p, 1 f, 1 r
 - 2008: 4 p, 2 f, 3 r
 - 2009: 1 p, 1 f, 0 r
 - 2010: 3 p, 1 f, 3 r
 - 2011: 5 p, 0 f, 4 r
 - 2012: 3 p, 0 f, 2 r
 - 2014: 2 p, 1 f, 0 r
 - 2015: 2 p, 0 f, 2 r
 - 2016: 1 p, 0 f, 1 r
 - 2017: 1 p, 0 f, 1 r
 - 2018: 3 p, 0 f, 2 r

**Comment by chriscool** [comment]
@jnareb if you want more statistics, let me know and I will help as I can.

**Comment by sivaraam** [comment]
Just for the note, I'm gathering information about contributors who've contributed via GSoC. I've put [a draft MR](https://github.com/git/git.github.io/pull/69) with the information that I've gathered so far. Feel free to comment on the same.

**Comment by chriscool** [comment]
@sivaraam I left some comments, thanks!
